src/Altinn.Notifications.Email.Integrations/Consumers/KafkaConsumerBase.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Altinn.Notifications.Email.Integrations.Configuration;
+
+using Confluent.Kafka;
+
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Altinn.Notifications.Integrations.Kafka.Consumers;
+
+/// <summary>
+/// Base class for Kafka consumers
+/// </summary>
+public abstract class KafkaConsumerBase<T> : BackgroundService
+    where T : class
```

Comment:
Type T is never actually used anywhere. Something we forgot?

Reply:
Yes, this is in use. The logger is dependent on T, and we also use it to for logging to know which consumer logged something, rather than it comming from this generic class